sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.1.0 (Unreleased)
```

Comment:
we will release it immediately after merging it to the main branch, so that's why we changed everywhere. But for the future feature we will take into account your comments, thanks. @maximrytych-ms can you pls mention Victor's comments in our android wiki

Reply:
@maximrytych-ms we should also go with incrementing a patch number as it's only bugfix

Reply:
Ok. Fixed.